docs/administration.platforms.md

```diff
@@ -1,37 +1,70 @@
 ---
 id: administration.platforms
-title: Platform Operation
+title: meshPlatforms
 ---
 
-Operators can perform various maintainance operations for their platforms. For maintenance of Landing Zones, please see
-[Landing Zone Administration](meshcloud.landing-zones.md).
+A key task for Platform Operators is integrating your Cloud Platforms as meshPlatforms into meshStack. This enables
+tenant creation, replication and metering of your Cloud Platforms. The maintenance includes
+general information about the Cloud Platform as well as structuring and configuring them.
```

Comment:
I don't really get this sentence: "The maintenance [...] configuring them". What are you trying to convey here?